common/shlibs

```diff
@@ -3875,3 +3875,11 @@ libhidrd_opt.so.0 hidrd-0.2.0_1
 libhidrd_strm.so.0 hidrd-0.2.0_1
 libhidrd_fmt.so.0 hidrd-0.2.0_1
 libjcat.so.1 libjcat-0.1.2_1
+
```

Comment:
Why a space?

Reply:
An oversight from a previous rebase; fixed in the latest push. Thanks for catching this.